pkg/resourcemanager/config/config.go

```diff
@@ -33,10 +33,28 @@ var (
 	podNamespace           string
 	targetNamespaces       []string
 	secretNamingVersion    secrets.SecretNamingVersion
+	operatorMode           OperatorMode
 
 	testResourcePrefix string // used to generate resource names in tests, should probably exist in a test only package
 )
 
+// OperatorMode determines whether we'll run watchers and/or webhooks.
+type OperatorMode string
+
+const (
+	OperatorModeBoth     = OperatorMode("both")
+	OperatorModeWebhooks = OperatorMode("webhooks")
+	OperatorModeWatchers = OperatorMode("watchers")
+)
+
+func (m OperatorMode) IncludesWebhooks() bool {
```

Comment:
Would this be more cleanly done with an enum that was an `int` done as a bitmask where `Includes(mode OperatorMode)` bool?

Reply:
My initial reaction is that I don't think so, but I'm just going to try it that way to see.

Reply:
It came out pretty nice!

Reply:
D'oh, I rebased this PR onto `upstream/master` from home yesterday without pulling from `origin/operator-mode` first (which would have brought in the enum change). All fixed now.